README.md

```diff
@@ -22,6 +22,17 @@ yarn && yarn deploy
 
 The cli will prompt for approval on IAM Roles and Permissions twice in the full deploy. Once for the backend stack and then again for the client stack. The cli will prompt for an email. After the deploy is complete, an email will be sent to address provided with credentials for logging in.
 
+2. CICD Deploy:
+
+There is also a way to deploy the solution that invokes the same CICD pipeline that is used by Solutions Builder team. This will create a separate stack that loads all resource onto CodePipeline, and then uses CodePipeline to invoke CDK. Make sure stack->region inside package.json contains its original value, "%%REGION%%".
+
+```
+./deployment/build_and_deploy_project.sh [bucket-name-minus-region] [version] [email address]
+```
```

Comment:
2 questions here

* What is the purpose to have region mentioned in the bucket name? Is the naming convention really required? 

* If naming convention is required, I woukld prefer the customer not take the additional effort to edit the string before passing it. I find that manual editing is more error prone than a simple copy-paste. Could we could keep the bucket name as is and strip the regex to extract the region and evaluate if it matches the configured region.

Reply:
The answer to the first question is, as far as I can tell, yes. The region is necessary because different buckets are used for holding the project in different regions.

However, this may not be necessary for our project (the SB team is changing things to make use of CDK properly) and I would like to hear from the Solutions Builder tech lead about it. I think we should hear from him before we put in the extra regex stripping work, and if he says that it is still necessary, then we can make the changes as you suggest (i agree about manual errors).

For all other demos in the meantime, people should just use "yarn && yarn deploy" to deploy the solution, which does not use this part.